test-vectors/did_dht/create.json

```diff
@@ -117,8 +112,7 @@
               "crv": "secp256k1",
               "x": "1_o0IKHGNamet8-3VYNUTiKlhVK-LilcKrhJSPHSNP0",
               "y": "qzU8qqh0wKB6JC_9HCu8pHE-ZPkDpw4AdJ-MsV2InVY",
-              "alg": "ES256K",
-              "kid": "0GkvkdCGu3DL7Mkv0W1DhTMCBT9-z0CkFqZoJQtw7vw"
```

Comment:
so what was the rationale for removing these `kid`'s in this file?

Reply:
the `kid` is now not populated by default and computed whenever it is necessary. the test vector (previously using JWK) assumed that it would be populated by default so the test would fail. 

same for alg - alg also isn't populated by default, since crv (a required field) is more accurate anyway.